spec/features/user_register_and_pay_with_card_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+feature 'user register and pay with card', {js: true, vcr: true} do 
```

Comment:
You could use RSpec tagging feature and `{js: true, vcr: true}` replace with `:js, :vcr`